test/delivery_methods/database_test.rb

```diff
@@ -17,7 +17,7 @@ class WithDelayedDatabaseDelivery < Noticed::Base
     end
 
     assert_equal 1, user.notifications.count
-    assert_equal :bar, user.notifications.last.params[:foo]
+    assert_equal :bar, user.notifications.last.params[:foo].to_sym
```

Comment:
This should deserialize as a symbol automatically, so we probably also need the symbol serializer.

Reply:
Ah yes, good point.